snowshu/core/replica/replica_factory.py

```diff
@@ -39,6 +40,8 @@ def _execute(self,
             self.config.name = name
 
         graph.build_graph(self.config)
+        cross_db_relations = list(graph.graph.nodes) if graph.graph is not None and \
+            not isinstance(graph.graph, Mock) else None
```

Comment:
The application code shouldn't need to branch for testing. As `Mock` isn't used internally to snowshu, it shouldn't be handled here.

Also, if this method fixes the problem, we shouldn't have any cross db references outside of the `if self.config.source_profile.adapter.SUPPORTS_CROSS_DATABASE` block. This variable should be renamed.